save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/Forum.kt

```diff
@@ -191,7 +193,7 @@ private fun ChildrenBuilder.renderLeftColumn(
                         }
                         div {
                             className = ClassName("col-12 text-center text-xs")
-                            +rating.toString()
+                            +(rating ?: 0L).toString()
```

Comment:
shouldn't this `0L` be the default value for rating itself and not null?

Reply:
Probably, but it was not me who designed it. Non-nullable rating might be more clear